test/unit/convertor.test.js

```diff
@@ -283,7 +283,8 @@ describe('Logical Groups', function () {
     });
 });
 
-describe('Relations', function () {
+// @TODO: what to do with these unit tests?
+describe.skip('Relations', function () {
```

Comment:
think we can throw whole 'Relations' suite away as the rest of tests 'Many-To-Many' etc.. are more specific and cover this ground anyway

Reply:
agree. will remove for now 👍